.github/workflows/verify.yml

```
@@ -67,11 +67,11 @@ jobs:
           - 2.7
           - 3.0.3
         test_cmd:
-          - bundle exec rake rspec-rerun:spec SPEC_OPTS="--tag content"
-          - bundle exec rake rspec-rerun:spec SPEC_OPTS="--tag ~content"
+          - SPEC_OPTS="--tag content --tag ~acceptance" bundle exec rake rspec-rerun:spec
+          - SPEC_OPTS="--tag ~content --tag ~acceptance" bundle exec rake rspec-rerun:spec
           # Used for testing the remote data service
-          - bundle exec rake rspec-rerun:spec SPEC_OPTS="--tag content" REMOTE_DB=1
-          - bundle exec rake rspec-rerun:spec SPEC_OPTS="--tag ~content" REMOTE_DB=1
+          - SPEC_OPTS="--tag content --tag ~acceptance" REMOTE_DB=1 bundle exec rake rspec-rerun:spec
+          - SPEC_OPTS="--tag ~content --tag ~acceptance" REMOTE_DB=1 bundle exec rake rspec-rerun:spec
```

Let's revert these, so they're opt-in, rather than opt-out. We can rely on CI to be the final check here

```suggestion
          - SPEC_OPTS="--tag content" REMOTE_DB=1 bundle exec rake rspec-rerun:spec
          - SPEC_OPTS="--tag ~content" REMOTE_DB=1 bundle exec rake rspec-rerun:spec
```